### ISSUE TITLE
Implement offset between encoders 0 and true north 0

### ISSUE THREAD

**Issue by arathuun** [YCPRadioTelescope/YCP-RT-ControlRoom#106]
The telescope will not have the encoders 0 be true north 0, thus we need to have an offset for when we move the telescope, such that a point 0 degrees is correct.

**Comment by jminor717** [comment]
there will likely need to be 2 offsets, one between the 0 defined by the home sensor and the 0 of the encoders, then the offset addressed in this issue. these should be kept as 2 separate values since the offset between the telescope and north will not change once the telescope is installed. the other offset between the home and encoder will be determined at run time by the home method in the PLC. coincidentally this is one of the bits of data that i wanted to store in a local database independently of the backend

**Comment by AktionP** [comment]
This may have been completed already by Jake, so we will need to get in contact with him to see what he has done, if anything.

If there has been nothing done, this issue will likely be deleted as the mechanical team is currently on hold.

**Comment by jminor717** [comment]
no this is not completed since the offset will need to change once the telescope is in place. but it shouldn't be deleted so that we can track the issue until that happens.

**Comment by pdnelson** [comment]
The second offset Jake mentioned is now covered in #301. #301 should be completed before this one, so that we have the encoder offset working.
